cmake/OpenCVConfig.cmake

```diff
@@ -84,9 +84,9 @@ elseif(MINGW)
   set(OpenCV_RUNTIME mingw)
 
   execute_process(COMMAND ${CMAKE_CXX_COMPILER} -dumpmachine
-                  OUTPUT_VARIABLE OPENCV_GCC_TARGET_MACHINE
+                  OUTPUT_VARIABLE CMAKE_OPENCV_GCC_TARGET_MACHINE
```

Comment:
I think the old name is more appropriate, and you should fix the other occurrence instead. We shouldn't put things into the `CMAKE_` "namespace" in a config file.